cicd/main.yml

```diff
@@ -0,0 +1,110 @@
+#trigger:
+#  batch: true
+#  branches:
+#    include:
+#      - refs/heads/master
+#  paths:
+#    exclude:
+#    - README.md
+
+pool: Computer-Vision-Team-Amsterdam
```

Comment:
im using the shared agent pool, maybe here is does not matter since we don't do much azure stuff

Reply:
I think it cannot pull the model from AS if I don't use ours